diff --git a/node_modules/@candlefinance/slider/src/index.tsx b/node_modules/@candlefinance/slider/src/index.tsx
index 96909b7..2562d46 100644
--- a/node_modules/@candlefinance/slider/src/index.tsx
+++ b/node_modules/@candlefinance/slider/src/index.tsx
@@ -30,9 +30,11 @@ export interface props {
   ballIndicatorHeight?: number;
   ballIndicatorPosition?: number;
   ballIndicatorTextColor?: string;
+  snowballValue?: number|string;
 }
 
 const Slider = ({
+  snowballValue,
   width = 350,
   height = 30,
   borderRadius = 25,
@@ -62,6 +64,7 @@ const Slider = ({
   >(undefined);
 
   React.useEffect(() => {
+    console.log('snowballValue',snowballValue)
     const panResponder = PanResponder.create({
       onStartShouldSetPanResponder: () => true,
       onMoveShouldSetPanResponder: () => false,
@@ -208,7 +211,8 @@ const Slider = ({
               },
             ]}
           >
-            {Math.round(_value.current * 100) / 100}
+            {/* {Math.round(_value.current * 100) / 100} */}
+{snowballValue}
           </Text>
         </Animated.View>
       )}
